qkeras/utils.py

```diff
@@ -178,7 +178,7 @@ def quantize_activation(layer_config, activation_bits):
 
 def get_config(quantizer_config, layer, layer_class, parameter=None):
   """Returns search of quantizer on quantizer_config."""
-  quantizer = quantizer_config.get(layer["name"],
+  quantizer = quantizer_config.get(layer["config"]["name"],
```

Comment:
we are ok with it. Just want to check your rationale about layer["config"]["name"], instead of layer["name"]. basically, they are the same. Any corner case you have encountered?